src/main/java/org/prebid/server/bidder/coinzilla/CoinzillaBidder.java

```diff
@@ -0,0 +1,87 @@
+package org.prebid.server.bidder.coinzilla;
+
+import com.iab.openrtb.request.BidRequest;
+import com.iab.openrtb.response.BidResponse;
+import com.iab.openrtb.response.SeatBid;
+import io.vertx.core.MultiMap;
+import io.vertx.core.http.HttpMethod;
+import org.apache.commons.collections4.CollectionUtils;
+import org.prebid.server.bidder.Bidder;
+import org.prebid.server.bidder.model.BidderBid;
+import org.prebid.server.bidder.model.BidderError;
+import org.prebid.server.bidder.model.HttpCall;
+import org.prebid.server.bidder.model.HttpRequest;
+import org.prebid.server.bidder.model.Result;
+import org.prebid.server.json.DecodeException;
+import org.prebid.server.json.JacksonMapper;
+import org.prebid.server.proto.openrtb.ext.response.BidType;
+import org.prebid.server.util.HttpUtil;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+public class CoinzillaBidder implements Bidder<BidRequest> {
+
+    private static final String OPENRTB_VERSION = "2.5";
+
+    private final String endpointUrl;
+    private final JacksonMapper mapper;
+
+    public CoinzillaBidder(String endpointUrl, JacksonMapper mapper) {
+        this.endpointUrl = HttpUtil.validateUrl(Objects.requireNonNull(endpointUrl));
+        this.mapper = Objects.requireNonNull(mapper);
+    }
+
+    @Override
+    public Result<List<HttpRequest<BidRequest>>> makeHttpRequests(BidRequest request) {
+        if (CollectionUtils.isEmpty(request.getImp())) {
```

Comment:
You can remove this, [since](https://github.com/prebid/prebid-server-java/blob/master/src/main/java/com/iab/openrtb/request/BidRequest.java#L33)